api-controllers/ServiceBrokerApiController.js

```diff
@@ -196,7 +196,7 @@ class ServiceBrokerApiController extends FabrikBaseController {
             }
           });
         } else {
-          return eventmesh.apiServerClient.patchOSBResource({
+          return eventmesh.apiServerClient.updateOSBResource({
```

Comment:
will it take care of context being patched?

Reply:
Why do we want to patch context? Why not overwrite the context object?

Reply:
We want to patch context as delete call does not send context and we do not want to loose the context overwriting it.

Reply:
**What it will do**
Old CRD is
```javascript
spec:
  context:
    org_guid: xx
    platform: cf
```
If Patch call has context: {org_guid: yy}
New CRD will become
```javascript
spec:
  context:
    org_guid: yy
```

If Patch call doesn't have context
New CRD doesn't change context field
```javascript
spec:
  context:
    org_guid: xx
    platform: cf
```

Reply:
hold on, if the spec is having no context object, will it keep the context?

Reply:
apiserver patch call doesn't change the fields which are not provided in patch call AFAIK. Tested . It works 😄

Reply:
cool.